tekton/get-version.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-versions
+spec:
+  resources:
+    inputs:
+    - name: source
+      type: git
+  steps:
+  - name: pull
+    image: goreleaser/goreleaser
+    workingDir: /workspace/source
+    script: |
+      git status; git fetch -p --all
+  - name: get-versions
+    image: goreleaser/goreleaser # because it has git
+    workingDir: /workspace/source
+    script: |
+      echo -n $(git tag --points-at HEAD) > /tekton/results/version
+      echo -n $(git tag --points-at HEAD | sed 's/^.//' | awk -F '[.]' '{ print $1}') > /tekton/results/major
```

Comment:
perhaps shorter `echo -n $(git tag --points-at HEAD | sed 's/^v\([^\.]*\).*/\1/'`) > `

your command (and mine) would get the `0` of `0.8.1`, is that what you want ? don't you want something like 0.8 instead? and `1` being the minor?

Reply:
In the end, in results I want `v0.8.0` (version), `0` (major), `8` (minor), in the result files.

Reply:
so you can do : 

major:
> git tag --points-at HEAD | sed 's/^v\([^\.]*\)\.\([^\.]*\).*/\1/'

minor:

> git tag --points-at HEAD | sed 's/^v\([^\.]*\)\.\([^\.]*\).*/\2/'